app/views/home/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= form_for @search_developer_form, url: '/' do |f| %>
+  <%= f.select :programming_language, ProgrammingLanguage.all.collect {|pl| [pl.name, pl.id]}, prompt: 'Select Developer Programming', name: 'Developer Programming' %>
```

Comment:
- As for `collect`, refer to the following.
https://github.com/bbatsov/ruby-style-guide#map-find-select-reduce-size
- You can use `pluck` here, can't you?
Read through the following page.
https://github.com/jp-ryuji/playbook/wiki/Useful-Methods

---

Please read through the following style guides (Ruby and Rails), too.
There're Vietnamese versions.
https://github.com/jp-ryuji/playbook/wiki/Coding-Style-Guide

Reply:
@jp-ryuji (Y)